ironic_bug_dashboard/configs/ironic.json

```diff
@@ -0,0 +1,34 @@
+{
+    "projects": [
+        "bifrost",
+        "ironic",
+        "ironic-inspector",
+        "ironic-lib",
+        "ironic-prometheus-exporter",
+        "ironic-python-agent",
+        "ironic-python-agent-builder",
+        "ironic-ui",
+        "metalsmith",
+        "networking-baremetal",
+        "networking-generic-switch",
+        "nova",
```

Comment:
This should not be here: it causes all 1000+ nova bugs to get loaded. It's enough to have it in tagged_projects.